cdap-hbase-compat-0.96/src/main/java/co/cask/cdap/data2/util/hbase/HBase96TableUtil.java

```diff
@@ -84,14 +85,10 @@ public boolean hasNamespace(HBaseAdmin admin, String namespace) throws IOExcepti
   }
 
   @Override
-  public void createNamespaceIfNotExists(HBaseAdmin admin, String namespace) throws IOException {
-    Preconditions.checkArgument(admin != null, "HBaseAdmin should not be null");
+  public void createNamespaceIfNotExists(HBaseDDLExecutor ddlExecutor, String namespace) throws IOException {
```

Comment:
Can we remove this method and just call the ddlExecutor's method? It doesn't do anything at this point.